src/main/java/com/graphhopper/converter/ConverterConfiguration.java

```diff
@@ -44,6 +49,7 @@
     private boolean opencagedata;
     private boolean pelias;
     private boolean nettoolkit = true;
+    private boolean photon = true;
```

Comment:
Why are the defaults different?

Reply:
What do you mean? Why some variables are initialized with true and some with no value?

Reply:
Yes, why we enabled nettoolkit but not e.g. opencagedata.

Reply:
I was wondering about that as well. I think this grew over time with different contributors, for example Opencagedata probably committed this differently than Nettoolkit. Should I added a commit to make them all equal. The main resource that controls these variables is the config.yml, so I added it there as well.

Reply:
Ok, thanks!